charts/rayjob/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for rayjob.
```

Comment:
I can see that some values used in the `rayjob.yaml` did not show up in this file. Would you like to update the `values.yaml` file to include all values used.

Reply:
Ok, I will updata the values.yaml file.